CMakeLists.txt

```diff
@@ -109,6 +109,22 @@ if(MSVC)
       endif(${flag_var} MATCHES "/MD")
     endforeach(flag_var)
   endif()
+  # Disable common MSVC warnings
+  # Integer conversion warnings(e.g. int64 to int)
+  add_compile_options(/wd4244)
```

Comment:
we might revisit to cmake file to avoid global flags, but for now it is good enough